src/insns/acperm_32bit.adoc

```diff
@@ -40,6 +40,8 @@ These cases are defined to return useful minimal sets of permissions, which may
 +
 NOTE: Future extensions may allow more combinations of permissions, especially for MXLEN=64.
 
+All rules are applied in the order shown on this page.
+
```

Comment:
I guess the main problem is that the RV32 rules need to be inserted at the right location. Maybe we should change this to a list that includes all of them in order and prefix the RV32 ones with `RV32-only:...`

Reply:
ok - we've reworked the rules - please have a look